### ISSUE TITLE
convert and migrate icecast

### ISSUE THREAD

**Issue by butlerx** [redbrick/nix-configs#6]
An icecast server is running on paphos for dcufm 

docs for current setup https://docs.redbrick.dcu.ie/services/icecast2

**Comment by benmcmahon100** [comment]
Lil hot tip for anyone intending on looking at this - icecast2 **DOES NOT** like reverse proxies. It may work okay being exposed to the outside world via NAT but I've never personally tested it - I.E. the best way to set this up as for now is to test streaming to it using port forwarding so as far as icecast will be concerned it will be facing the broadcast source directly

**Comment by benmcmahon100** [comment]
Just some notes re my previous comment:
 - https://forum.netgate.com/topic/124773/issue-connecting-to-server-behind-firewall-from-outside-solved/11
 - http://www.icecast.org/docs/icecast-trunk/relaying/
 - https://stackoverflow.com/questions/38120135/how-to-view-the-correct-client-ip-in-icecast-stats-when-icecast-works-behind-th
 - https://wiki.xiph.org/index.php?title=Icecast_Server/known_reverse_proxy_restrictions&mobileaction=toggle_view_desktop
 - http://lists.xiph.org/pipermail/icecast-dev/2015-March/002451.html
 - http://lists.xiph.org/pipermail/icecast/2004-April/007088.html

TL;DR: seem to be adding support in the next minor ver (2.5)
